iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -539,8 +546,6 @@ struct TileAndDistributeOnTensorsPattern
     pullInProducersInSameGroup(rewriter, dispatchOp, shapedOperands,
                                tiledLinalgOp, rootOpAttr.getInt());
 
-    tiledLinalgOp.op.getOperation()->removeAttr(kRootOpAttr);
```

Comment:
The idea of a root op is only relevant to this pass. The attribute should be removed. Nothing downstream should rely on this being a root operation for fusion.